contracts/DevTokensHolder.sol

```
@@ -80,7 +80,7 @@ contract DevTokensHolder is Owned {
         }
 
         collectedTokens = collectedTokens.add(canExtract);
-        if (!snt.transfer(owner, canExtract)) throw;
+        assert(snt.transfer(owner, canExtract));
```

I contend that none of these functions should be returning a value in the first place, they should all either do what the function name implies or fail hard (throw).

Having functions that return 0 on failure introduces opportunities for bugs/security issues because the caller not only needs to know to check the result, but also needs to remember to do it everywhere and not get the conditional backwards.